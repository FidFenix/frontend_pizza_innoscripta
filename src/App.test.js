import React from 'react';
import { create } from "react-test-renderer";

import FooterComp from  './components/footer/footer.component';
import HeaderComp from  './components/header/header.component';

/*it('render correctly HeaderComp component', () => {  
  const comp = create(<HeaderComp />).toJSON();
  expect(comp).toMatchSnapshot();
});*/

it('render correctly FooterComp component', () => {  
  const comp = create(<FooterComp />).toJSON();
  expect(comp).toMatchSnapshot();
});
