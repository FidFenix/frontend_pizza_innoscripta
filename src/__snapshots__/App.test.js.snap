// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly FooterComp component 1`] = `
<div
  className="footer-content"
>
  <ul>
    <li>
      About
    </li>
    <li>
      Contact us
    </li>
    <li>
      Created by Fidel Mamani
    </li>
  </ul>
</div>
`;

exports[`render correctly HeaderComp component 1`] = `
<div
  className="footer-content"
>
  <ul>
    <li>
      About
    </li>
    <li>
      Contact us
    </li>
    <li>
      Created by Fidel Mamani
    </li>
  </ul>
</div>
`;
